functions/completed-checks/v1.js

```diff
@@ -93,7 +93,7 @@ async function handleCompletedCheck (context, completedCheckMessage) {
     PartitionKey: completedCheckMessage.checkCode,
     RowKey: uuid(),
     eventType: 'completed-check',
-    payload: JSON.stringify(completedCheckMessage),
+    payload: '<removed>',
```

Comment:
maybe this could take the compressed message 🤔

Reply:
The payload is already stored in the DB, which is surely the source of truth?

Reply:
good point.  this is merely a trace of activity, so should be fine as is 👍